docs/products/metrics/howto/migrate-m3db-thanos.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Migrate Aiven for M3DB to Aiven for Metrics
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Aiven for M3DB databases to Aiven for Thanos Metrics using the Aiven Console migration tool.
+
+## Compatibility considerations
+
+Review your current M3DB setup before migrating to Aiven for Metrics.
+
+- **Review database setup:** Examine your M3DB database's data structures,
+  storage patterns, and configurations. Identify any unique features, custom settings,
+  and specific configurations.
+- **API and query language compatibility:** Both Thanos Metrics and M3DB use PromQL,
+  so most existing queries will work as is. However, be aware that the same metric
+  might have a slightly different name when written directly to Thanos compared to its
+  name in M3DB. For example, `prometheus_<metric-name>` in M3DB might be called
+  `<metric-name>` in Thanos. Consequently, after migration, you will have two names
```

Comment:
Probably should phrase it a bit differently. Something along the line  "might have" instead of "will have". This depends on the metric.

Reply:
Thanks! Done.